pom.xml

```diff
@@ -1349,6 +1359,45 @@
   </build>
 
   <profiles>
+    <!-- hadoop version: mvn validate -Phadoop-3.3 ,spark still use hadoop2.7.2 by default, More details please check SPARK-23534   -->
+    <profile>
+      <id>hadoop-3.3</id>
+      <properties>
+        <hadoop.version>3.3.1</hadoop.version>
+        <curator.version>4.2.0</curator.version>
+        <hadoop-hdfs-client.artifact>hadoop-hdfs-client</hadoop-hdfs-client.artifact>
+      </properties>
+    </profile>
+    <!-- hadoop version: mvn validate -Phadoop-2.7  -->
+    <profile>
+      <id>hadoop-2.7</id>
+      <properties>
+        <hadoop.version>2.7.2</hadoop.version>
+        <curator.version>2.7.1</curator.version>
+      </properties>
+    </profile>
+    <!-- spark2-hadoop3 version:spark2.4 shade use hadoop2.7.2 by default mvn validate -Pspark-2.4-hadoop-3.3  -->
+    <profile>
+      <id>spark-2.4-hadoop-3.3</id>
```

Comment:
this profile should be moved to spark's pom.xml,  and this profile should work with -Phadoop-3.3.   and put the linkis-hadoop-hdfs-client-shade dependency to this profile.